CONTRIBUTING.md

```diff
@@ -96,6 +97,12 @@ When multiple versions coexist, **terms are only tracked in their English versio
 
 > We intend to expand coverage, but we focus for the moment on this subset of documents to fine-tune the system.
 
+### `executeClientScripts`
+
+This boolean property, when set to `true` executes client script before trying to get document content by browsing to the URL with an headless browser. It's set to `false` by default. It is useful when the content of the document is loaded or modified dynamically on the browser side by client scripts.
```

Comment:
```suggestion
In some cases, the content of the document is only loaded (or is modified dynamically) by client scripts.
When set to `true`, this boolean property loads the page in a headless browser to load all assets and execute client scripts before trying to get document content.
Since the performance cost of this approach is high, it is set to `false` by default, relying on the HTML content only.
```